docs/Getting Started/index.rst

```diff
@@ -19,4 +19,5 @@ documentation <https://pthree.org/2012/04/17/install-zfs-on-debian-gnulinux/>`__
    NixOS <https://nixos.wiki/wiki/NixOS_on_ZFS>
    openSUSE/index
    RHEL and CentOS
```

Comment:
Sorry, I wasn't clear,
- remove it from ToC here
- add `:orphan:` on the first line of `RHEL and CentOS.rst` page to exclude it from `WARNING: document isn't included in any toctree` checks

Reply:
Thanks for the notice, fixed.